.travis.yml

```diff
@@ -36,6 +36,7 @@ matrix:
         - sudo make install prefix=/usr
         - make testinstall prefix=/usr
         - popd
+      env: PATH="/usr/lib/llvm-3.7/bin:$PATH"
```

Comment:
From working on the bindgen WIP this is needed to pick LLVM 3.7 and not the one provided by Travis.